src/modules/site/templates/tools/heritability/list-all.html

```diff
@@ -0,0 +1,117 @@
+{% extends "_layouts/default.html" %}
+
+{% block custom_head %}
+    <link rel="stylesheet" href="//cdn.datatables.net/1.10.21/css/jquery.dataTables.min.css" />
```

Comment:
we are loading a different version of databables JS + CSS in `head.html`. Can you remove this dependency here and use the global? Otherwise, if you need this version specifically we may need to refactor the global one into the templates where it is actually used.